improve vuln cpe matching on macos (#6985)
fleetdm/fleet: docs/Deploying/Configuration.md

##### cpe_database_url

> URL to fetch the CPE dictionary database from. Some users want to control where Fleet gets its database from.

Reviewer: ```suggestion
You can fetch the CPE dictionary database from this URL. Some users want to control where Fleet gets its database.
```

When Fleet sees this value defined, it downloads the file directly.

> It expects a file in the same format as can be found in https://github.com/fleetdm/nvd/releases.

Reviewer: ```suggestion
It expects a file in the same format that can be found in https://github.com/fleetdm/nvd/releases.
```

  	cpe_database_url: ""
  ```

##### cpe_translations_url

URL to fetch the CPE translations from.
Translations are used when matching software to CPE entries in the CPE database that would otherwise be missed for various reasons.
When Fleet sees this value defined, it downloads the file directly.
It expects a file in the same format as can be found in https://github.com/fleetdm/nvd/releases.

> If this value is not defined, Fleet checks for the latest release in Github and only downloads it if needed.

Reviewer 2: Did you mean https://github.com/fleetdm/nvd/blob/master/cpe_translations.json? (does it check `/releases`?)

Reviewer 3: There was[ a thread](https://github.com/fleetdm/fleet/pull/6985#discussion_r933658334) about moving those artifacts (both the `cpe_translations.json` and the `oval_sources.json` files) to `/releases` - but I think that's outside the scope of this PR and maybe we should do it in a separate issue.

Reviewer 3: Ah NVM my prev. comment - seems like this will be included in this PR

  	cpe_database_url: ""
  ```

##### cpe_translations_url

> URL to fetch the CPE translations from.

Reviewer: ```suggestion
You can fetch the CPE translations from this URL.
```

Translations are used when matching software to CPE entries in the CPE database that would otherwise be missed for various reasons.
When Fleet sees this value defined, it downloads the file directly.

> It expects a file in the same format as can be found in https://github.com/fleetdm/nvd/releases.

Reviewer: ```suggestion
It expects a file in the same format that can be found in https://github.com/fleetdm/nvd/releases.
```

If this value is not defined, Fleet checks for the latest release in Github and only downloads it if needed.

- Default value: `""`
- Environment variable: `FLEET_VULNERABILITIES_CPE_TRANSLATIONS_URL`
- Config file format:
  ```
  vulnerabilities:
  	cpe_translations_url: ""
  ```

##### cve_feed_prefix_url

> Similarly to the CPE dictionary, we allow users to define where to get the CVE feeds from.

Reviewer: ```suggestion
Like the CPE dictionary, we allow users to define where to get the CVE feeds.
```

> In this case, the url should be a host that serves the files in the path /feeds/json/cve/1.1/.

Reviewer: ```suggestion
In this case, the URL should be a host that serves the files in the path /feeds/json/cve/1.1/.
```

> Fleet expects to find there all the JSON Feeds that can be found in https://nvd.nist.gov/vuln/data-feeds.

Reviewer: ```suggestion
Fleet expects to find all the JSON Feeds that can be found in https://nvd.nist.gov/vuln/data-feeds.
```
